app/src/main/java/ch/epfl/sdp/fragment/UserInfectionFragment.java

```diff
@@ -159,6 +183,11 @@ private void executeHealthStatusChange() {
         }
     }
 
+    private void letFirebaseEnjoyMyRecovery() {
```

Comment:
I appreciate some humor when reading a PR but for our future selves you should rename that function

Reply:
done